### PR TITLE
Masurca

### PR DESCRIPTION
**Pull request by xfzhao**
updating

### CODE REVIEW

File: var/spack/repos/builtin/packages/masurca/package.py [hunk 1]
```diff
@@ -12,10 +12,26 @@ class Masurca(Package):
        approaches."""
 
     homepage = "http://www.genome.umd.edu/masurca.html"
-    url      = "ftp://ftp.genome.umd.edu/pub/MaSuRCA/latest/MaSuRCA-3.2.3.tar.gz"
+    version('3.3.1', sha256='587d0ee2c6b9fbd3436ca2a9001e19f251b677757fe5e88e7f94a0664231e020',
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/masurca/package.py]: I think it would be sufficient to just add 3.3.1, and leave the prior versions as-is

Comment (xfzhao) [reply var/spack/repos/builtin/packages/masurca/package.py]: Remove those 3 extra files.

Can not download the data from the old ftp server for 3.2.3. ( ftp://ftp.genome.umd.edu/pub/MaSuRCA/latest) 

the earliest version in github is 3.2.4.

**Comment by baberlevi** [comment]
Confirmed - old FTP site is not available, so original packages can no longer be retrieved. I cut the version list down to the most current, and the last 3.2.X version available since the original ones were not present on github (as you said).  Confirmed no other packages depend on masurca, so I think this change seems safe.

File: var/spack/repos/builtin/packages/supernova/package.py [hunk 2]
```diff
@@ -1,45 +0,0 @@
-# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/supernova/package.py]: This still needs fixed - we don't want this change to delete the file from the repo

**Pull request merged**